packages/flutter/lib/src/animation/tween.dart

```diff
@@ -68,6 +78,8 @@ class Tween<T extends dynamic> extends Animatable<T> {
   T lerp(double t) => begin + (end - begin) * t;
 
   /// Returns the interpolated value for the current value of the given animation.
+  ///
+  /// This method returns `begin` and `end` when the animation values are 0.0 or 1.0, respectively.
```

Comment:
Also, note that 0.0 and 1.0 should be lowerBound and upperBound, respectively (which are, by default, 0.0 and 1.0). I just saw this in the source code and it was incorrect in the original doc.